code/style.css

```diff
@@ -1,4 +1,207 @@
+/* GLOBAL STYLES */
+
 html {
-  background: #0025ff;
-  color: #fff;
+  box-sizing: border-box;
 }
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  background-color: rgb(232, 240, 242);
+  color: rgb(5, 55, 66)
+}
+
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding-bottom: 0;
+  background-color: rgb(5, 55, 66, .90);
+  color: white;
+}
+
+h1 {
+  font-weight: normal;
+  font-size: 18px;
+  margin-left: 10px;
+}
+
+h2 {
+  text-align: center;
+  font-weight: normal;
+  font-size: 18px;
+}
+
+i {
+  font-size: x-large;
+}
+
+/* HEADER & NAVIGATION */
+.logo-container {
+  display: flex;
+  align-items: center;
+  margin-left: 10px;
+  padding-top: 5px;
+}
+
+.logo {
+  width: 60px;
+  }
+  
+.hidden-anchor {           /* the hidden anchor is needed to keep the margin between the fixed header and the first big-news-article*/
```

Comment:
You leave comments for yourself and describe what these specific line of code does. AWESOME! :)